datastore/cloud-client/snippets.py

```diff
@@ -991,8 +991,8 @@ def index_merge_queries(client):
 def main(project_id):
     client = datastore.Client(project_id)
 
-    for name, function in globals().iteritems():
-        if name in ("main", "defaultdict") or not callable(function):
+    for name, function in list(globals().items()):
```

Comment:
I think it was not necessary to convert this into `list`. Was there a reason for doing that?

```suggestion
    for name, function in globals().items():
```